packages/cadl-vs/src/VSExtension.cs

```diff
@@ -189,31 +196,36 @@ private static string GetDevelopmentCadlServerPath()
         }
 #endif
 
-        private (string, string[], IDictionary<string, string>) resolveCadlServer(IWorkspaceSettings? settings)
+        private (string, string[], IDictionary<string, string>) resolveCadlServer()
         {
             var env = new Dictionary<string, string>();
             var args = new string[] { "--stdio" };
 #if DEBUG
             // Use local build of cadl-server in development (lauched from F5 in VS)
             if (InDevelopmentMode())
             {
-               var options = Environment.GetEnvironmentVariable("CADL_SERVER_NODE_OPTIONS");
-               var module = GetDevelopmentCadlServerPath();
-               return ("node.exe", new string[] { module, options }.Concat(args).ToArray(), env);
+                var options = Environment.GetEnvironmentVariable("CADL_SERVER_NODE_OPTIONS");
```

Comment:
hhm did the formatter not catch this before?

Reply:
It breaks with #if it seems, I ran VS format command :(